tl/controllers/middleware.rst

```diff
@@ -129,121 +129,120 @@ scripts, that add middleware::
             $middlewareQueue->add(new ContactPluginMiddleware());
         });
 
-PSR-7 Requests and Responses
+Ang PSR-7 na mga Hiling at mga Tugon
 ============================
 
-Middleware and the new HTTP stack are built on top of the `PSR-7 Request
-& Response Interfaces <http://www.php-fig.org/psr/psr-7/>`__. While all
-middleware will be exposed to these interfaces, your controllers, components,
-and views will *not*.
+Ang Middleware at ang bagong HTTP stack ay itinayo sa ibabaw ng `PSR-7 Request
+& Response Interfaces <http://www.php-fig.org/psr/psr-7/>`__. Habang lahat
+ng middleware ay malantad sa mga interfaces na ito, iyong mga controller, mga components
+at mga view ay *hindi*.
 
-Interacting with Requests
+Pakikipag-ugnay sa mga Hiling
 -------------------------
 
-The ``RequestInterface`` provides methods for interacting with the headers,
-method, URI, and body of a request. To interact with the headers, you can::
+Ang ``RequestInterface`` ay nagbibigay ng mga pamamaraan para sa pag-uugnay sa mga header,
+pamamaraan, URI, at katawan ng isang hiling. Para makapag-ugnay sa mga header, kaya mo::
```

Comment:
- Please translate request to kahilingan, instead of hiling
- maaari mong::

Reply:
done change